README.md

```diff
@@ -1 +1,49 @@
-# HelloMicroverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# HTML and CSS Simple project
+
+> This is a simple HTML and CSS project with linters included there.
+
+
+## Built With
+
+- HTML
+- CSS
+
+
+## Getting Started
+To get a local copy up and running:
+1. Clone this repository or download the Zip folder:
+**``git clone repo [<link>](https://github.com/habasm/HelloMicroverse.git)``**
+1. Navigate to the location of the folder in your machine:
+**``you@your-Pc-name:~$ cd <folder>``**
+
+## Authors
+
+👤 Habitamu Asimare
+
+- GitHub: [@habasm](https://github.com/habasm)
+- Twitter: [@AsimareHabitamu](https://twitter.com/AsimareHabitamu)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] [OPTIONAL] Please remove all the unnecessary information here in order to make your readme more professinal :+1: